share/spack/docker/leap-15.dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM opensuse/leap:15.2
+MAINTAINER Christian Goll <cgoll@suse.com>
+
+ENV DOCKERFILE_BASE=opensuse          \
+    DOCKERFILE_DISTRO=opensuse_leap   \
+    DOCKERFILE_DISTRO_VERSION=15.2    \
+    SPACK_ROOT=/opt/spack      \
+    DEBIAN_FRONTEND=noninteractive    \
+    CURRENTLY_BUILDING_DOCKER_IMAGE=1 \
+    container=docker
+
+COPY bin   $SPACK_ROOT/bin
+COPY etc   $SPACK_ROOT/etc
+COPY lib   $SPACK_ROOT/lib
+COPY share $SPACK_ROOT/share
+COPY var   $SPACK_ROOT/var
+
+RUN ln -s $SPACK_ROOT/share/spack/docker/entrypoint.bash \
+          /usr/local/bin/docker-shell \
+ && ln -s $SPACK_ROOT/share/spack/docker/entrypoint.bash \
+          /usr/local/bin/interactive-shell \
+ && ln -s $SPACK_ROOT/share/spack/docker/entrypoint.bash \
+          /usr/local/bin/spack-env
+RUN mkdir -p $SPACK_ROOT/opt/spack
```

Comment:
Could you move these copies and symlinks so that they come after the package installs and manpage cleanup?  We do something similar in the other builds to avoid reinstalling packages every time something in Spack changes.

Reply:
Also, boto3 isn't installed in this image, so Spack will not be able to use mirrors or binary caches over S3.  There may be an argument that can be made that we shouldn't include this in the image, but if/until there is consensus on the matter, it should probably be included to match what is on the other images.

Reply:
spack is now copied into the container after the package install and boto3 is installed via the system package.
I am also calling spack explicitly as this works better for me, when building the container with buildah